static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -386,7 +504,14 @@ export function QuickContextHoverWrapper(props: ContextProps) {
         showUnderline
         delay={HOVER_DELAY}
         header={getHoverHeader(props.dataRow, props.contextType)}
-        body={getHoverBody(api, props.dataRow, props.contextType, props.organization)}
+        body={getHoverBody(
+          api,
+          props.dataRow,
+          props.contextType,
+          props.organization,
+          props.location,
+          props.projects
```

Comment:
If you destructure all of your props in the definition on line 490, we can avoid repeating `props.` so much

Reply:
Yeah, will do.